src/index.js

```diff
@@ -41,6 +41,7 @@ const createRunner = ({ runtime = "worker_threads" } = {}) =>
       this._pool = new (runInBand ? InBandTinypool : Tinypool)({
         filename: new URL("./worker-runner.js", import.meta.url).href,
         runtime,
+        minThreads: maxWorkers,
```

Comment:
Should we only add this if it's `!runInBand && _isProcessRunner`?

Reply:
I guess it doesn't matter.
Since `WorkerThread` does not use `pMap`, it should not need this parameter, and there is no harm even if it does.

Reply:
There seems also `idleTimeout` option.

On piscina docs https://github.com/piscinajs/piscina#constructor-new-piscinaoptions